runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterConfig.java

```diff
@@ -25,13 +25,15 @@
 package net.runelite.client.plugins.hunter;
 
 import java.awt.Color;
+import net.runelite.client.config.Alpha;
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
 @ConfigGroup("hunterplugin")
 public interface HunterConfig extends Config
 {
+	@Alpha
```

Comment:
None of this has any effect due to the overlay hardcoding an alpha value of 100.